File: namsor/kotlin/src/main/kotlin/oseg/namsor_examples/SubclassificationFullExample.kt
package oseg.namsor_examples

import app.namsor.client.infrastructure.*
import app.namsor.client.apis.*
import app.namsor.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class SubclassificationFullExample
{
    fun subclassificationFull()
    {
        ApiClient.apiKey["X-API-KEY"] = "YOUR_API_KEY"

        try
        {
            val response = PersonalApi().subclassificationFull(
                countryIso2 = "NG",
                fullName = "Jannat Rahmani",
            )

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling PersonalApi#subclassificationFull")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling PersonalApi#subclassificationFull")
            e.printStackTrace()
        }
    }
}

File: namsor/typescript-node/src/SubclassificationFullExample.ts
import * as fs from 'fs';
import api from "namsor_client"
import models from "namsor_client"

const apiCaller = new api.PersonalApi();
apiCaller.setApiKey(api.PersonalApiApiKeys.api_key, "YOUR_API_KEY");

apiCaller.subclassificationFull(
  "NG", // countryIso2
  "Jannat Rahmani", // fullName
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling PersonalApi#subclassificationFull:");
  console.log(error.body);
});
